docs/en_US/guide/ai/python_openvino_tutorial.md

```diff
@@ -0,0 +1,286 @@
+2. Create a Python entry file that exposes all the implemented interfaces. Because multiple extensions can be implemented in a single plugin, you need an entry file that defines the implementation classes for each extension. The content is a main function, which is the entry point for the plugin runtime. It calls the methods in the SDK to define the plugin, including the plugin name, and a list of keys for the source, sink, and function implemented in the plugin. Here only a function plugin named `inference` is implemented, and its corresponding implementation method is `inferenceIns`. The Python plug-in process is independent of the eKuiper main process.
+
+    ```python
+    if __name__ == '__main__':
+        # Define the plugin
+        c = PluginConfig("defect", {}, {},
+                         {"inference": lambda: inferenceIns})
+        # Start the plugin instance
+        plugin.start(c)
+    ```
+
+3. Create a plugin description file in JSON format to define the metadata of the plugin. The file name must be the same as the plugin name, i.e. `defect.json`. The function names defined therein must correspond exactly to the entry file, and the contents of the file are as follows. Where, executable is used to define the name of the plugin's executable entry file.
+
+    ```json
+    {
+      "version": "v1.0.0",
+      "language": "python",
+      "executable": "main.py",
+      "sources": [
+      ],
+      "sinks": [
+      ],
+      "functions": [
+        "inference"
+      ]
+    }
+    ```
+
+At this point we have completed the development of the plugin, next we just need to package all the files in the directory into a zip file. The zip file should have a file structure similar to:
+
+- inference.py
+- requirements.txt
+- install.sh
+- main.py
+- defect.json
+- models
+  - model.bin
+  - model.xml
+- functions
+  - defect.json
+
+## Plugin Installation
+
+As the same with installing native plugins, we can also install Python plugins via the eKuiper manager UI or the REST API. To use the REST API, upload the zip file packaged above to the machine where eKuiper is located. Then use the following API to install it:
+
+```text
+### Install pyai plugin
+POST http://{{host}}/plugins/functions
+Content-Type: application/json
+
+{"name":"defect", "file": "file:///tmp/defect.zip"}
+```
+
+The installation process requires an Internet connection to download dependencies, including `ffmpeg libsm6 libxext6`, which may take a long time depending on the network conditions.
+
+## Run the plugin
+
+Once the plugin installed, we can use it in our rule. We will create a rule to receive image byte data from a mqtt topic and do the inference for the image by OpenVINO model.
```

Comment:
```suggestion
After installing the plugin, it can be incorporated into our rule. We'll create a rule that receives image byte data from an MQTT topic and performs inference on the image using the OpenVINO model.
```